lib/MusicBrainz/Server/Data/Release.pm

```diff
@@ -172,12 +180,22 @@ sub _where_filter
         }
         my $country_id_filter = $filter->{country_id};
         my $date_filter = $filter->{date};
-        if (defined $country_id_filter || defined $date_filter) {
+        if (defined $country_id_filter && $country_id_filter eq '-1'
+            && !defined $date_filter) {
+            push @query, '(NOT EXISTS (SELECT 1 FROM release_event re WHERE re.release = release.id) OR EXISTS (SELECT 1 FROM release_event re WHERE re.release = release.id AND country IS NULL))';
+        } elsif (defined $country_id_filter || defined $date_filter) {
+            if ($country_id_filter eq '-1' && !defined $date_filter) {
+            } else {
+            }
```

Comment:
Both branches seem to be empty here?

Reply:
Oh, that was from when I thought I needed to query `release_unknown_country`, but then I realized it's all the view. Will drop.